WooCommerce/WooCommerceTests/Extensions/IconsTests.swift

```diff
@@ -71,6 +71,10 @@ final class IconsTests: XCTestCase {
         XCTAssertNotNil(UIImage.deleteImage)
     }
 
+    func testDeleteCellImageIconIsNotNil() {
+        XCTAssertNotNil(UIImage.deleteCellImage)
```

Comment:
Since the icon has a custom size, what do you think about adding also a test that checks the size of the image?👍 

```
func testDeleteCellImageIconDefaultSize() {
        let deleteCellImage = UIImage.deleteCellImage
        XCTAssertEqual(deleteCellImage.size, CGSize(width: 22, height: 22))
    }
```

Reply:
nice suggestion! added in https://github.com/woocommerce/woocommerce-ios/pull/2505/commits/60177e57082e3947d5a4da86f323dbb56e11e810